lib/src/AnyRpcServer.cpp

```diff
@@ -32,10 +32,12 @@ AnyRpcServer::AnyRpcServer(int anyrpcPort)
         "Click on the object at the given path | mouseClick(string path)",
         [this](std::string path) { mouseClick(std::move(path)); });
 
-    utils::AddFunctionToAnyRpc<void(std::string, int)>(methodManager, "mouseClickWithButton",
-        "Click on the object at the given path with the given mouse button | mouseClickWithButton(string path, int "
-        "mouseButton)",
-        [this](std::string path, int mouseButton) { mouseClick(std::move(path), mouseButton); });
+    utils::AddFunctionToAnyRpc<void(std::string, int, int)>(methodManager, "mouseClickWithButton",
+        "Click on the object at the given path with the given mouse button and Control Keys | "
+        "mouseClickWithButton(string path, int mouseButton)",
+        [this](std::string path, int mouseButton, int keyModifier = KeyModifiers::None) {
```

Comment:
I don't think the default parameter works for AnyRPC and the rpc method always expects three arguments.
```suggestion
        "mouseClickWithButton(string path, int mouseButton, int keyModifier)",
        [this](std::string path, int mouseButton, int keyModifier) {
```

Reply:
I think you know more about AnyRPC than I do. As a Python developer, I think default parameters are cool.